# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_yodlee
# 
# Translators:
# Martin Trigaux, 2020
# Arma Gedonsky <armagedonsky@hot.ee>, 2020
# Eneli Õigus <enelioigus@gmail.com>, 2020
# Martin Aavastik <martin@avalah.ee>, 2020
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~12.5+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-08-26 08:04+0000\n"
"PO-Revision-Date: 2019-08-26 09:34+0000\n"
"Last-Translator: Martin Aavastik <martin@avalah.ee>, 2020\n"
"Language-Team: Estonian (https://www.transifex.com/odoo/teams/41243/et/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: et\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "(Diagnostic: %r for URL %s)"
msgstr "(Diagnostika: %r URL-ile%s)"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Account"
msgstr "Konto"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Already Registered (436):Your Auto-Registration could not be "
"completed because the site reports that your account is already registered."
"  Please log in to the site to confirm and then complete the site addition "
"process with the correct login information."
msgstr ""
"Konto on juba registreeritud (436): teie automaatset registreerimist ei "
"õnnestunud lõpule viia, kuna leht andis märku, et teie konto on juba "
"registreeritud. Palun logi lehele, et kinnitada ja seejärel lõpetada lehe "
"lisamise protsess korrektse sisselogimise informatsiooniga."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Locked (407): We could not update your account because it appears "
"your account has been locked. This usually results from too many "
"unsuccessful login attempts in a short period of time. Please visit the site"
" or contact its customer support to resolve this issue.  Once done, please "
"update your account credentials in case they are changed."
msgstr ""
"Konto on lukus (407): meil ei õnnestunud uuendada teie kontot, kuna teie "
"konto on lukus. Ilmselt on keegi või olete ise teinud liiga palju kordi "
"sisselogimiskatseid lühikese perioodi jooksul. Palun külastage lehte või "
"võtke probleemi lahendamiseks ühendust klienditeenindusega."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Not Found(408): We're sorry, we couldn't find any accounts for you "
"at the site. Please log in at the site and confirm that your account is set "
"up, then try again."
msgstr ""
"Kontot ei leitud (408): meil on kahju, kuid meil ei õnnestunud teie kontot "
"leida. Palun logi lehele sisse ja kinnita, et sinu konto on seadistatud, "
"seejärel proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Reported Closed (422): We were unable to update your account "
"information because it appears one or more of your related accounts have "
"been closed.  Please deactivate or delete the relevant account and try "
"again."
msgstr ""
"Konto on suletud (422): meil ei õnnestu teie konto informatsiooni uuendada, "
"kuna näib, et üks või mitu teie kontot on suletud. Palun deaktiveerige või "
"kustutage seotud konto ja proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Account Type Not Supported(417):The type of account we found is not "
"currently supported.  Please remove this site and add as a  manual account."
msgstr ""
"Konto tüüp pole toetatud (417): leitud konto tüüpi hetkel ei toetata. Palun "
"eemaldage see leht ja lisage konto käsitsi."

#. module: account_yodlee
#: model:ir.model.fields,help:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Active/Inactive on Yodlee system"
msgstr "Aktiivne/mitteaktiivne Yodlee süsteemis"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "An Error has occurred (code %s)"
msgstr "Tekkis probleem (kood %s)"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_res_company
msgid "Companies"
msgstr "Ettevõtted"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Could not make sense of parameters: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (402): We could not update your account "
"because your username and/or password were reported to be incorrect.  Please"
" re-verify your username and password."
msgstr ""
"Volitus on vaja üle kontrollida (402): meil ei õnnestunud teie kontot "
"uuendada, kuna teie kasutajanimi ja/või parool on valed. Palun kinnitage "
"uuesti oma kasutajanimi ja parool."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Credential Re-Verification Required (420):The site has merged with another. "
"Please re-verify your credentials at the site and update the same."
msgstr ""
"Uuesti tõendamine nõutud (420): leht on ühendatud teisega. Palun kinnitage "
"lehe volitused uuesti ja värskendaga seda."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Error %s, message: %s, reference code: %s"
msgstr "Viga %s, sõnum: %s, viitekood: %s"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Home Value Not Found (511): The site is unable to provide home value for "
"your property. We suggest you to delete this site."
msgstr ""
"Kodu väärtust ei leitud (511): lehel ei õnnestu pakkuda kodu väärtust teie "
"varale. Me sooviteme teil kustutada see leht."

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Invalid Language Setting (421): The language setting for your site account "
"is not English. Please visit the site and change the language setting to "
"English."
msgstr ""
"Kehtetud keeleseaded (421): teie lehe konto keel pole inglise keel. Palun "
"külasta lehte ja muuda keel inglise keeleks."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Additional Authentication Enrollment Required (521) : Please "
"enroll in the new security authentication system, <Account Name> has "
"introduced. Ensure your account settings in <Cobrand> are updated with this "
"information."
msgstr ""
"MFA viga: nõutud on täiendava autentimise registreerimine (521): palun "
"sisesta uus turvalisuse autentimise süsteem, <Account Name> on kasutusele "
"võetud. Veenduge, et teie konto seadeid <Cobrand> on uuendatud selle "
"informatsiooniga."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Expired (524):We're sorry, the "
"authentication information you provided has expired. Please try again."
msgstr ""
"MFA viga: autentimise teave on aegunud (524): vabandame, kuid teie "
"autentimise informatsioon on aegunud. Palun proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (520):We're sorry, the site "
"indicates that the additional authentication information you provided is "
"incorrect. Please try updating your account again."
msgstr ""
"MFA viga: autentimise teave on vale (520): vabandame, aga leht viitab "
"sellele, et teie autentimise informatsioon on vale. Palun proovige uuesti "
"oma kontot uuendada!"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."
msgstr ""
"MFA viga: autentimise informatsioon on vale (523): vabandame, kuid teie "
"esitatud autentimise informatsioon on vale. Palun proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Required (519): Your account was not "
"updated as your authentication information like security question and answer"
" was unavailable or incomplete. Please update your account settings."
msgstr ""
"MFA viga: autentimise informatsioon on nõutud (519): teie kontot ei "
"värskendatud, sest teie autentimisandmed nagu turvaküsimused ja vastused, "
"olid kättesaamatud või puudulikud. Palun uuenda oma konto seadeid."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Authentication Information Unavailable (518):Your account was not"
" updated as the required additional authentication information was "
"unavailable. Please try now."
msgstr ""
"MFA viga: autentimise informatsioon pole saadaval (518): teie kontot ei "
"uuendatud, kuna nõutav täiendav autentimise informatsioon ei olnud "
"kättesaadav. Palun proovige nüüd uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Credential Re-Verification Required (526): We could not update "
"your account because your username/password or additional security "
"credentials are incorrect. Please try again."
msgstr ""
"MFA viga: nõutud on volituste uuesti kontrollimine (526): meil ei õnnestunud"
" uuendada teie kontot, kuna teie kasutanimi/parool või täiendavad "
"turvavolitused on valed. Palun proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Problem Updating Account (525): We could not update your account "
"for technical reasons. This type of error is usually resolved in a few days."
" We apologize for the inconvenience. Please try again later."
msgstr ""
"MFA viga: konto uuendamise probleem (525): tehnilistel põhjustel meil ei "
"õnnestunud uuendada teie kontot. Seda tüüpi viga lahendatakse tavaliselt "
"mõne päeva jooksul. Vabandame tekkinud segaduste pärast. Palun proovige "
"hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Request Timed Out (522) :Your request has timed out as the "
"required security information was unavailable or was not provided within the"
" expected time. Please try again."
msgstr ""
"MFA viga: taotlus on aegunud (522): teie taotlus on aegunud, kuna nõutud "
"turvateave ei olnud kättesaadav või ei olnud kättesaadav oodatud aja "
"jooksul. Palun proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"MFA error: Site Device Information Expired(509): We're sorry, we can't "
"update your account because your token is no longer valid at the site. "
"Please update your information and try again, or contact customer support."
msgstr ""
"MFA viga: lehe seadme informatsioon on aegunud (509): vabandame, kuid meil "
"ei õnnestu teie kontot uuendada, kuna teie lehe \"token\" ei ole enam "
"kehtiv. Palun uuendage oma informatsiooni ja proovige uuesti või võtke "
"ühendust klienditeenindusega."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Multiple User Logins(416): We attempted to update your account, but another "
"session was already established at the same time.  If you are currently "
"logged on to this account directly, please log off and try after some time"
msgstr ""
"Mitme kasutaja sisselogimine (416): püüdsime uuendada teie kontot, kuid "
"teine sessioon loodi samal ajal. Kui te olete hetkel kontol sisse logitud, "
"siis palun logige välja ja proovige mõne aja pärast."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"New Login Information Required(506):We're sorry, to log in to this site, you"
" need to provide additional information. Please update your account and try "
"again."
msgstr ""
"Uus sisselogimisteave on nõutud (506): vabandame, kuid lehele logimiseks "
"peate esitama lisainformatsiooni. Palun uuendage oma kontot ja proovige "
"uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Account Found (423): We were unable to detect an account. Please verify "
"that you account information is available at this time and If the problem "
"persists, please contact customer support at online@odoo.com for further "
"assistance."
msgstr ""
"Kontot ei leitud (423): me ei suutnud leida kontot. Palun kinnitage, et teie"
" konto informatsioon on hetkel kättesaadav ning kui probleem püsib, siis "
"palun võtke ühendust klienditeenindusega e-kirja teel online@odoo.com."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"No Payees Found(512):Your request cannot be completed as no payees were "
"found in your account."
msgstr ""
"Maksesaajat ei leitud (512): teie taotlust ei saa lõpule viia, kuna teie "
"kontol ei leitud maksesaajat."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Online institution"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Please configure an email in the company settings."
msgstr "Palun seadistage e-posti aadress ettevõtte seadete alt."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (406): We could not update your account because the"
" site requires you to perform some additional action. Please visit the site "
"or contact its customer support to resolve this issue. Once done, please "
"update your account credentials in case they are changed else try again."
msgstr ""
"Konto uuendamise probleem (406): me ei saanud teie kontot uuendada, kuna "
"leht nõuab täiendavaid toiminguid. Palun külastage lehte või probleemi "
"lahendamiseks võtke ühendust klienditeenindusega. Kui tehtud, siis uuendage "
"oma konto volitusi, kui need on muutunud, muul juhul proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account (517): We are sorry, there was a technical problem "
"updating your account. Please try again."
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(401):We're sorry, your request timed out. Please "
"try again."
msgstr ""
"Probleem konto uuendamisel (401): teie taotlus aegus. Vabandame! Palun "
"proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(403):We're sorry, there was a technical problem "
"updating your account. This kind of error is usually resolved in a few days."
" Please try again later."
msgstr ""
"Probleem konto uuendamisel (403): vabandame, kuid teie konto uuendamisel "
"tekkis tehniline probleem. Sellist tüüpi viga lahendatakse tavaliselt mõne "
"päeva jooksul. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(404):We're sorry, there was a technical problem "
"updating your account. Please try again later."
msgstr ""
"Probleem konto uuendamisel (404): vabandame, kuid konto uuendamisel tekkis "
"tehniline probleem. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(409): We could not update your account because the "
"end site is experiencing technical difficulties."
msgstr ""
"Probleem konto uuendamisel (409): meil ei õnnestunud teie kontot uuendada, "
"kuna lõpp-lehel on tehnilisi probleeme."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(412): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Probleem konto uuendamisel (412): meil ei õnnestunud uuendada teie kontot "
"kuna lehel on tehnilisi probleeme."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(413):We're sorry, we couldn't update your account "
"at the site because of a technical issue. This type of problem is usually "
"resolved in a few days. Please try again later."
msgstr ""
"Probleem konto uuendamisel (413): vabandame, kuid meil ei õnnestunud "
"tehniliste probleemide tõttu teie kontot uuendada. Sellist tüüpi probleem "
"lahendatakse tavaliselt mõne päeva jooksul. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(415): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"Probleem konto uuendamisel (415): meil ei õnnestunud teie kontot uuendada, "
"kuna lehel on tehnilisi probleeme."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(418): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Probleem konto uuendamisel (418): meil ei õnnestunud uuendada teie kontot "
"kuna lehel on tehnilisi probleeme. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(419):We're sorry, we couldn't update your account "
"because of unexpected variations at the site. This kind of problem is "
"usually resolved in a few days. Please try again later."
msgstr ""
"Probleem konto uuendamisel (419): vabandame, kuid meil ei õnnestunud "
"ootamatute erinevuste tõttu teie kontot uuendada. Sellist tüüpi probleem "
"lahendatakse tavaliselt mõne päeva jooksul. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(425): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"Probleem konto uuendamisel (425): meil ei õnnestunud uuendada teie kontot "
"kuna lehel on tehnilisi probleeme. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(426): We could not update your account for "
"technical reasons. This type of error is usually resolved in a few days. We "
"apologize for the inconvenience."
msgstr ""
"Probleem konto uuendamisel (426): meil ei õnnestunud tehniliste probleemide "
"tõttu teie kontot uuendada. Sellist tüüpi probleem lahendatakse tavaliselt "
"mõne päeva jooksul. Vabandame ebamugavuste pärast!"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Problem Updating Account(507):We're sorry, Yodlee has just started providing"
" data updates for this site, and it may take a few days to be successful as "
"we get started. Please try again later."
msgstr ""
"Probleem konto uuendamisel (507): Yodlee alustas lehel andmete uuendamist "
"ning see võib võtta aega mõned päevad enne, kui saame edukalt alustada. "
"Vabandame! Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Property Record Not Found (510): The site is unable to find any property "
"information for your address. Please verify if the property address you have"
" provided is correct."
msgstr ""
"Vara kirjet ei leitud (510): lehel ei õnnestunud leida teie vara "
"omandiinformatsioon. Palun kontrollige, et antud vara aadress oleks "
"korrektne."

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr "Pakkuja tüüp"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "Online konto sünkroniseerimise pakkuja"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-Verification Required (429): We could not update your account due to the "
"site requiring you to verify your personal information. Please log in to the"
" site and update the fields required."
msgstr ""
"Uuesti üle kontrollimine nõutud (429): Meil ei õnnestunud teie kontot "
"uuendada, kuna teie kontol on vaja personaalne informatsioon üle "
"kontrollida. Palun logige lehele ja uuendage nõutud väljad."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (427): We could not update your account due to the "
"site requiring you to view a new promotion. Please log in to the site and "
"click through to your account overview page to update the account.  We "
"apologize for the inconvenience."
msgstr ""
"Uuesti üle kontrollimine nõutud (427): Meil ei õnnestunud teie kontot "
"uuendada, kuna leht nõuab uue reklaami vaatamist. Palun logige lehele ja "
"vajutage konto ülevaate lehele, et uuendada kontot. Me vabandame tekkinud "
"ebamugavuste pärast!"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Re-verification Required (428): We could not update your account due to the "
"site requiring you to accept a new Terms & Conditions. Please log in to the "
"site and read and accept the T&C."
msgstr ""
"Uuesti üle kontrollimine nõutud (428): Meil ei õnnestunud teie kontot "
"uuendada, kuna enne tuleb nõustuda uute nõuete ja tingimustega. Palun logige"
" lehele ja nõustuge nõuete ja tingimustega."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (433): Auto registration is not complete. "
"Please complete your registration at the end site. Once completed, please "
"complete adding this account."
msgstr ""
"Registreerimine nõuab tähelepanu (433): automaatne registreerimine ei ole "
"täielik. Palun viige oma registreerimine lõpuni lehe lõpus. Kui lõpetatud, "
"siis saate lõpetada selle konto lisamise."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (434): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Registreerimine nõuab tähelepanu (434): automaatne registreerimine ei ole "
"täielik ning vajab teilt täiendavat sisendinfot. Palun kinnitage uuesti oma "
"registreerimise informatsioon, et lõpetada registreerimisprotsess."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Registration Requires Attention (435): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"Registreerimine nõuab tähelepanu (435): automaatne registreerimine ei ole "
"täielik ning vajab teilt täiendavat sisendinfot. Palun kinnitage uuesti oma "
"registreerimise informatsioon, et lõpetada registreerimisprotsess."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Request Timed Out (508): We are sorry, your request timed out due to "
"technical reasons. Please try again."
msgstr ""
"Taotlus on aegunud (508): vabandame, aga teie taotlus aegus tänu "
"tehnilistele põhjustele. Palun proovige uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Requested Account Type Not Found (414): We could not find your requested "
"account. You may have selected a similar site under a different category by "
"accident in which case you should select the correct site."
msgstr ""
"Nõutavat konto tüüpi ei leitud (414): meil ei õnnestunud leida teie soovitud"
" kontot. Võimalik, et valisite ekslikult sarnase lehe erineva kategooria alt"
" ning peaksite valima uuesti õige lehe."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Server not reachable, please try again later"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Down for Maintenance(424):We were unable to update your account as the "
"site is temporarily down for maintenance. We apologize for the "
"inconvenience.  This problem is typically resolved in a few hours. Please "
"try later."
msgstr ""
"Lehel toimub hooldus (424): me ei saa uuendada teie kontot, kuna leht on "
"ajutiselt maas. Vabandame tekkinud ebamugavuste pärast! Selline probleem "
"laheneb tavaliselt paari tunni jooksul. Palun proovige hiljem uuesti."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Available (411):The site no longer provides online services "
"to its customers.  Please delete this account."
msgstr ""
"Leht pole enam saadaval (411): See leht ei paku enam klientidele online "
"teenuseid. Palun kustutage see konto."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site No Longer Supported (430):This site is no longer supported for data "
"updates. Please deactivate or delete your account. We apologize for the "
"inconvenience."
msgstr ""
"Lehte enam ei toetata (430): seda lehte enam ei toetata andmete "
"uuendamisega. Palun deaktiveerige või kustutage oma konto. Vabandame "
"ebamugavuste pärast."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Site Not Supported (505): We currently does not support the security system "
"used by this site. We apologize for any inconvenience. Check back "
"periodically if this situation has changed."
msgstr ""
"Lehte ei toetata (505): me hetkel ei toeta selle lehe poolt kasutatavat "
"turvasüsteemi. Vabandame ebamugavuste pärast! Vaadake perioodiliselt uuesti,"
" kas olukord on muutunud."

#. module: account_yodlee
#: model:ir.actions.client,name:account_yodlee.yodlee_widget
msgid "Synchronization process completed"
msgstr ""

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "Timeout: the server did not reply within 30s"
msgstr "Pärin aegus: server ei vastanud 30 sekundi jooksul"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid ""
"Update Request Canceled(405):Your account was not updated because you "
"canceled the request."
msgstr ""
"Uuendamise taotlus on tühistatud (405): teie kontot ei uuendatud, kuna te "
"tühistasite taotluse"

#. module: account_yodlee
#: model:ir.model.fields.selection,name:account_yodlee.selection__account_online_provider__provider_type__yodlee
msgid "Yodlee"
msgstr "Yodlee"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Yodlee Account Status"
msgstr "Yodlee konto olek"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_status_code
msgid "Yodlee Status Code"
msgstr "Yodlee oleku kood"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_access_token
msgid "Yodlee access token"
msgstr "Yodlee juurdepääsukood"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_login
msgid "Yodlee login"
msgstr "Logi Yodlee'sse"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_password
msgid "Yodlee password"
msgstr "Yodlee parool"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_access_token
msgid "access_token"
msgstr "access_token"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:0
#, python-format
msgid "message: %s"
msgstr ""
